expr.go

```diff
@@ -127,6 +127,10 @@ func Function(name string, fn func(params ...interface{}) (interface{}, error),
 			Func:  fn,
 			Types: ts,
 		}
+		c.Types[name] = conf.Tag{
```

Comment:
@antonmedv You said that functions can come from two places: TypesTable and Function.
So here I made the wrong decision, putting delegate from `Function` to the `TypesTable`? Instead, I should use `config.Functions` map in the `config.Check` function, and where we are searching for suitable operator overload, right?

Reply:
Sounds right.